frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/EditTab/EditTab.tsx

```diff
@@ -70,43 +69,14 @@ function SelectedOptionList({
   component,
   handleComponentChange,
 }: SelectedOptionListProps) {
-  const { t } = useTranslation();
-
-  const handleDelete = () => {
-    if (component.options) {
-      delete component.options;
-    }
-
-    const emptyOptionsId = '';
-    handleComponentChange({
-      ...component,
-      optionsId: emptyOptionsId,
-    });
-
-    setComponentHasOptionList(false);
-  };
-
-  const label =
-    component.optionsId !== '' && component.optionsId !== undefined
-      ? component.optionsId
-      : t('ux_editor.modal_properties_code_list_custom_list');
-
   return (
-    <div aria-label={label} className={classes.chosenOptionContainer}>
+    <div className={classes.chosenOptionContainer}>
```

Comment:
could this css somehow be set on the OptionListEditor? And we could remove this abstraction?

Reply:
It can be moved to `OptionTabs.module.css` with altering already existing css there.